experimental/web/index.html

```diff
@@ -150,9 +170,8 @@
                       ) +
                       ') -> ' +
                       status +
-                      '\n  label: [' +
-                      label +
-                      ']';
+                      '\n  label: ' +
```

Comment:
It would probably make sense to write these as tagged string literals for readability. Eg `hi ${name}. That way we can omit the `+`s to join strings, and new lines are supported without `\n`

Reply:
Done.